### DISCUSSION TITLE
jpチャンネルを追加する

### DISCUSSION THREAD

**Discussion by Nobuchans** [iptv-org/community#596] [Q&A]
このURLを参考にして、全チャンネルを見られるようにしても。
（地上デジタル放送）https://ja.m.wikipedia.org/wiki/日本のテレビジョン放送局

（BS放送一覧）
101, 102| 日本放送協会 | NHK BS |
141, 142, 143| BS日本 | BS日テレ【BS4】
151、152、153 | BS朝日【BSA】
161、162、163 | BS-TBS
171, 172, 173| BSテレビ東京 | BSテレ東 |
181, 182, 183| ビーエスフジ | BSフジ |
191 | ワウワプライム
192 | ワウワライブ
193 | すごいシネマ
200 | | スター・チャンネル
211 | | 日本BS放送 | BS11 |
222 | BS12 トゥエルビ | TwellV
231, 232 | 放送大学学園 | 放送大学テレビ |
234 | グリーンチャンネル |
236 | アニマックス |
242 | Jスポーツ1
243 | Jスポーツ2
244 | Jスポーツ3
245 | Jスポーツ4
251 | 釣りビジョン | BS釣りビジョン
252 | WOWOWプラス | WOWOWプラス映画・ドラマ・スポーツ・音楽 |
255 | 日本映画放送 | 日本映画専門チャンネル |
256 | ディズニー・チャンネル | ディズニー |
260 | BS松竹東急 | 松竹東急 |
263 | ジャパネットブロードキャスティング | BSJapanext |
265 | BSよしもと
（CS放送一覧)
055	[ショップチャンネル]
218	[東映チャンネル]
219	[衛星劇場]	
296	[TBSチャンネル1 最新ドラマ・音楽・映画]
298	[テレ朝チャンネル]
299	テレ朝チャンネル2
317	[KBSワールド]
318	[Mnet]	
339	[ディズニージュニア]
349	[日テレNEWS24]
800	[スポーツライブ+]
801	[スカチャン1]
161	[QVC]
223	[映画・チャンネルNECO]
227	[ザ・シネマ]
240	ムービープラス
250	[スカイA]
254	[GAORA]	
257	[日テレジータス]
262	[ゴルフネットワーク]
290	[宝塚スカイステージ]
292	[時代劇専門チャンネル]
293	[ファミリーシアター]	
295	[MONDO TV]
297	TBSチャンネル2 名作ドラマ・スポーツ・アニメ
300	[日テレプラス ドラマ・アニメ・音楽ライブ]
301	[エンタメ〜テレ☆シネドラバラエティ]
305	[チャンネル銀河 歴史ドラマ・サスペンス・日本のうた]
307	[フジテレビONE スポーツ・バラエティ]
308	[フジテレビTWO ドラマ・アニメ] 
309	[フジテレビNEXT ライブ・プレミアム]
310	[スーパー！ドラマTV]
312	[Dlife]
314	[女性チャンネル♪LaLa TV]
322	[音楽・ライブ！スペースシャワーTV]
323	[MTV]
325	[MUSIC ON！ TV（エムオン！）]
330	[キッズステーション]
333	[アニメシアターX（AT-X）]
342	[ヒストリーチャンネル 日本・世界の歴史&エンタメ]
343	[ナショナルジオグラフィック]	
351	[TBSニュース] TBSニュースバード
294	[ホームドラマチャンネル 韓流・時代劇・国内ドラマ] 
311	[アクションチャンネル]
316	[ミステリーチャンネル]
321	[100%ヒッツ！スペースシャワーTV プラス]
324	[ミュージック・エア]
329	[歌謡ポップスチャンネル]
331	[カートゥーン ネットワーク 海外アニメ 国内アニメ]
340	[ディスカバリーチャンネル]
341	[アニマルプラネット]
353	[BBCニュース]
354	[CNNj]
363	囲碁・将棋チャンネル
191 WOWOW4[WOWOWシネマ]

[ＷＯＷＯＷシネマ](https://www.tvkingdom.jp/sp/chart/cs/400623.action)
[ＦＯＸムービー](https://www.tvkingdom.jp/sp/chart/cs/400624.action)
[スターチャンネル１ プレミアム](https://www.tvkingdom.jp/sp/chart/cs/400625.action)
[スターチャンネル２ セレクト](https://www.tvkingdom.jp/sp/chart/cs/400626.action)
[スターチャンネル３ 吹替専門](https://www.tvkingdom.jp/sp/chart/cs/400627.action)
[衛星劇場ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400628.action)
[東映チャンネルHD](https://www.tvkingdom.jp/sp/chart/cs/400629.action)
[イマジカＢＳ・映画](https://www.tvkingdom.jp/sp/chart/cs/400630.action)
[ザ・シネマHD](https://www.tvkingdom.jp/sp/chart/cs/400631.action)
[ムービープラスＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400632.action)
[映画・チャンネルNECO](https://www.tvkingdom.jp/sp/chart/cs/400633.action)
[日本映画専門チャンネルＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400634.action)
[Ｖ☆パラダイスＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400635.action)
[エキサイティング・グランプリ](https://www.tvkingdom.jp/sp/chart/cs/400636.action)
[FIGHTING TV サムライ](https://www.tvkingdom.jp/sp/chart/cs/400600.action)
[ゴルフネットワークHD](https://www.tvkingdom.jp/sp/chart/cs/400601.action)
[GAORA SPORTS](https://www.tvkingdom.jp/sp/chart/cs/400602.action)
[J SPORTS 1](https://www.tvkingdom.jp/sp/chart/cs/400603.action)
[J SPORTS 2](https://www.tvkingdom.jp/sp/chart/cs/400604.action)
[J SPORTS 3](https://www.tvkingdom.jp/sp/chart/cs/400606.action)
[J SPORTS 4](https://www.tvkingdom.jp/sp/chart/cs/400605.action)
[スカイA](https://www.tvkingdom.jp/sp/chart/cs/400607.action)
[日テレジータス](https://www.tvkingdom.jp/sp/chart/cs/400608.action)
[EXスポーツ](https://www.tvkingdom.jp/sp/chart/cs/400609.action)
[ダンスチャンネル by エンタメ〜テレ](https://www.tvkingdom.jp/sp/chart/cs/400665.action)
[ＢＳスカパー！](https://www.tvkingdom.jp/sp/chart/cs/400579.action)
[スカサカ！](https://www.tvkingdom.jp/sp/chart/cs/400580.action)
[スカチャン1](https://www.tvkingdom.jp/sp/chart/cs/400581.action)
[スカチャン2](https://www.tvkingdom.jp/sp/chart/cs/400582.action)
[スカチャン3](https://www.tvkingdom.jp/sp/chart/cs/400583.action)
[スカチャン4](https://www.tvkingdom.jp/sp/chart/cs/400584.action)
[スカチャン5](https://www.tvkingdom.jp/sp/chart/cs/400585.action)
[スカチャン6](https://www.tvkingdom.jp/sp/chart/cs/400586.action)
[スカチャン7](https://www.tvkingdom.jp/sp/chart/cs/400587.action)
[スカチャン8](https://www.tvkingdom.jp/sp/chart/cs/400588.action)
[スカチャン9](https://www.tvkingdom.jp/sp/chart/cs/400589.action)
[スカチャン10](https://www.tvkingdom.jp/sp/chart/cs/400590.action)
[スカチャン11](https://www.tvkingdom.jp/sp/chart/cs/400591.action)
[スカチャン12](https://www.tvkingdom.jp/sp/chart/cs/400592.action)
[スカチャン13](https://www.tvkingdom.jp/sp/chart/cs/400593.action)
[スカチャン14](https://www.tvkingdom.jp/sp/chart/cs/400594.action)
[スカパー！4K　総合](https://www.tvkingdom.jp/sp/chart/cs/400596.action)
[FOXスポーツ＆エンターテイメント](https://www.tvkingdom.jp/sp/chart/cs/400610.action)
[テレ朝チャンネル１　ドラマ・バラエティ・アニメ](https://www.tvkingdom.jp/sp/chart/cs/400611.action)
[フジテレビＮＥＸＴ　ライブ・プレミアム（ＨＤ）](https://www.tvkingdom.jp/sp/chart/cs/400613.action)
[フジテレビＯＮＥ スポーツ・バラエティ（ＨＤ）](https://www.tvkingdom.jp/sp/chart/cs/400614.action)
[フジテレビＴＷＯ　ドラマ・アニメ（ＨＤ）](https://www.tvkingdom.jp/sp/chart/cs/400615.action)
[ＴＢＳチャンネル１　最新ドラマ・音楽・映画１](https://www.tvkingdom.jp/sp/chart/cs/400616.action)
[エンタメ〜テレＨＤ☆シネドラバラエティ](https://www.tvkingdom.jp/sp/chart/cs/400618.action)
[日テレプラス　ドラマ・アニメ・音楽ライブ](https://www.tvkingdom.jp/sp/chart/cs/400619.action)
[ディズニー･チャンネル ディズニー映画･アニメ](https://www.tvkingdom.jp/sp/chart/cs/400620.action)
[ＷＯＷＯＷプライム](https://www.tvkingdom.jp/sp/chart/cs/400621.action)
[ＷＯＷOWライブ](https://www.tvkingdom.jp/sp/chart/cs/400622.action)
[ＴＢＳチャンネル２　名作ドラマ・スポーツ・アニメ](https://www.tvkingdom.jp/sp/chart/cs/400617.action)
[チャンネル銀河　歴史ドラマ・サスペンス・日本のうた](https://www.tvkingdom.jp/sp/chart/cs/400664.action)
[クラシカ・ジャパンHD](https://www.tvkingdom.jp/sp/chart/cs/400637.action)
[ミュージック・エア](https://www.tvkingdom.jp/sp/chart/cs/400638.action)
[ミュージック・ジャパンTV](https://www.tvkingdom.jp/sp/chart/cs/400639.action)
[MTV HD](https://www.tvkingdom.jp/sp/chart/cs/400640.action)
[MUSIC ON! TV（エムオン!）HD](https://www.tvkingdom.jp/sp/chart/cs/400641.action)
[音楽・ライブ！　スペースシャワーＴＶ](https://www.tvkingdom.jp/sp/chart/cs/400642.action)
[100％ヒッツ！スペースシャワーTV プラス HD](https://www.tvkingdom.jp/sp/chart/cs/400643.action)
[歌謡ポップスチャンネル](https://www.tvkingdom.jp/sp/chart/cs/400644.action)
[ミュージック・グラフィティＴＶ](https://www.tvkingdom.jp/sp/chart/cs/400645.action)
[スーパー！ドラマＴＶ ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400647.action)
[AXNミステリー HD](https://www.tvkingdom.jp/sp/chart/cs/400649.action)
https://www.tvkingdom.jp/sp/chart/cs/400650.action
[FOX HD](https://www.tvkingdom.jp/sp/chart/cs/400651.action)
[FOXクラシック 名作ドラマ](https://www.tvkingdom.jp/sp/chart/cs/400652.action)
[DATV](https://www.tvkingdom.jp/sp/chart/cs/400653.action)
[女性チャンネル♪LaLa TV](https://www.tvkingdom.jp/sp/chart/cs/400654.action)
[アジアドラマチックTV](https://www.tvkingdom.jp/sp/chart/cs/400655.action)
[ＫＢＳ Ｗｏｒｌｄ ](https://www.tvkingdom.jp/sp/chart/cs/400656.action)
[ＫＮＴＶ ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400657.action)
[Ｍｎｅｔ　ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400658.action)
[ヌーヴェルパラダイス](https://www.tvkingdom.jp/sp/chart/cs/400535.action)
[MONDO TV HD](https://www.tvkingdom.jp/sp/chart/cs/400659.action)
[ファミリー劇場ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400660.action)
[ホームドラマチャンネルＨＤ 韓流・時代劇・国内ドラマ](https://www.tvkingdom.jp/sp/chart/cs/400661.action)
[時代劇専門チャンネルHD](https://www.tvkingdom.jp/sp/chart/cs/400662.action)
[アイドル専門チャンネルＰｉｇｏｏ](https://www.tvkingdom.jp/sp/chart/cs/400663.action)
[Kawaiian TV](https://www.tvkingdom.jp/sp/chart/cs/400666.action)
[AT-X HD!](https://www.tvkingdom.jp/sp/chart/cs/400667.action)
[海外アニメ！カートゥーン ネットワーク](https://www.tvkingdom.jp/sp/chart/cs/400668.action)
[キッズステーション テレビアニメ･劇場版･ＯＶＡ](https://www.tvkingdom.jp/sp/chart/cs/400669.action)
[アニマックス HD](https://www.tvkingdom.jp/sp/chart/cs/400670.action)
[ディズニーＸＤ ギャグアニメ＆マーベル･ヒーロー](https://www.tvkingdom.jp/sp/chart/cs/400671.action)
[ナショジオ ワイルド HD](https://www.tvkingdom.jp/sp/chart/cs/400673.action)
[ヒストリーチャンネル 日本・世界の歴史＆エンタメ](https://www.tvkingdom.jp/sp/chart/cs/400674.action)
[ナショナル ジオグラフィック](https://www.tvkingdom.jp/sp/chart/cs/400675.action)
[ディスカバリーチャンネル](https://www.tvkingdom.jp/sp/chart/cs/400676.action)
[アニマルプラネット](https://www.tvkingdom.jp/sp/chart/cs/400677.action)
[ビジネス・ブレークスルー](https://www.tvkingdom.jp/sp/chart/cs/400557.action)
[ＳＯＲＡ−お天気チャンネル−](https://www.tvkingdom.jp/sp/chart/cs/400560.action)
[BBCワールドニュース](https://www.tvkingdom.jp/sp/chart/cs/400565.action)
[CNNｊ](https://www.tvkingdom.jp/sp/chart/cs/400566.action)
[CNN/US HD](https://www.tvkingdom.jp/sp/chart/cs/400567.action)
[中国テレビ★ＣＣＴＶ大富](https://www.tvkingdom.jp/sp/chart/cs/400568.action)
[日経CNBC](https://www.tvkingdom.jp/sp/chart/cs/400570.action)
[日テレNEWS24](https://www.tvkingdom.jp/sp/chart/cs/400571.action)
[TBSニュースバード](https://www.tvkingdom.jp/sp/chart/cs/400572.action)
[ベターライフチャンネル](https://www.tvkingdom.jp/sp/chart/cs/400529.action)
[パチンコ★パチスロＴＶ！](https://www.tvkingdom.jp/sp/chart/cs/400536.action)
[パチ・スロ　サイトセブンＴＶ](https://www.tvkingdom.jp/sp/chart/cs/400537.action)
[釣りビジョンHD](https://www.tvkingdom.jp/sp/chart/cs/400540.action)
[寄席チャンネル](https://www.tvkingdom.jp/sp/chart/cs/400542.action)
[旅チャンネルＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400544.action)
[鉄道チャンネル](https://www.tvkingdom.jp/sp/chart/cs/400546.action)
[囲碁・将棋チャンネルＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400521.action)
[ディズニージュニア お子さま向けディズニー番組](https://www.tvkingdom.jp/sp/chart/cs/400672.action)
[ライブ５３０](https://www.tvkingdom.jp/sp/chart/cs/400530.action)
[たちかわ・西武園競輪チャンネル](https://www.tvkingdom.jp/sp/chart/cs/400531.action)
[南関東地方競馬チャンネル](https://www.tvkingdom.jp/sp/chart/cs/400678.action)
[ＪＬＣ６８０ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400680.action)
[ＪＬＣ６８１ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400681.action)
[ＪＬＣ６８２ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400682.action)
[ＪＬＣ６８３ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400683.action)
[ＪＬＣ６８４ＨＤレディースチャンネル](https://www.tvkingdom.jp/sp/chart/cs/400684.action)
[ＪＬＣ６８５ＷプラスＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400685.action)
[ＪＬＣ６８６プラスＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400686.action)
[ＪＬＣ６８７プラスＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400687.action)
[グリーンチャンネルHD](https://www.tvkingdom.jp/sp/chart/cs/400688.action)
[グリーンチャンネル２ＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400689.action)
[スピードプラスワン６９５](https://www.tvkingdom.jp/sp/chart/cs/400695.action)
[ＴＶグローボ・インターナショナル](https://www.tvkingdom.jp/sp/chart/cs/400514.action)
[フェニックステレビ（鳳凰衛視）](https://www.tvkingdom.jp/sp/chart/cs/400518.action)
[ショップチャンネル](https://www.tvkingdom.jp/sp/chart/cs/400523.action)
[ジャパネットチャンネルＤＸ ハイビジョン](https://www.tvkingdom.jp/sp/chart/cs/400524.action)
[ＱＶＣ（キューヴィーシー）](https://www.tvkingdom.jp/sp/chart/cs/400525.action)
[ジュエリー☆ＧＳＴＶ](https://www.tvkingdom.jp/sp/chart/cs/400527.action)
[セレクトショッピング](https://www.tvkingdom.jp/sp/chart/cs/400528.action)
[ｋｍｐチャンネル](https://www.tvkingdom.jp/sp/chart/cs/400942.action)
[プレイボーイ チャンネルＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400943.action)
[レインボーチャンネルＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400944.action)
[ミッドナイト・ブルー](https://www.tvkingdom.jp/sp/chart/cs/400945.action)
[パラダイステレビＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400946.action)
[チェリーボムＨＤ](https://www.tvkingdom.jp/sp/chart/cs/400947.action)
[VENUS](https://www.tvkingdom.jp/sp/chart/cs/400957.action)
[バニラスカイチャンネル](https://www.tvkingdom.jp/sp/chart/cs/400958.action)
[エンタ！９５９](https://www.tvkingdom.jp/sp/chart/cs/400959.action)
[熟女・人妻専門 Zaptv](https://www.tvkingdom.jp/sp/chart/cs/400960.action)
[ダイナマイトTV](https://www.tvkingdom.jp/sp/chart/cs/400963.action)
[AV王](https://www.tvkingdom.jp/sp/chart/cs/400964.action)
[レッドチェリー](https://www.tvkingdom.jp/sp/chart/cs/400965.action)
[Splash](https://www.tvkingdom.jp/sp/chart/cs/400966.action)
[フラミンゴ](https://www.tvkingdom.jp/sp/chart/cs/400967.action)
[パワープラッツ](https://www.tvkingdom.jp/sp/chart/cs/400968.action)
[スカパー！プロモ599](https://www.tvkingdom.jp/sp/chart/cs/400599.action)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/new?template=4_channel-search.yml
